### ISSUE TITLE
react_on_rails 8.0.0 not generating production builds by default

### ISSUE THREAD

**Issue by serodriguez68** [shakacode/react_on_rails#875]
I upgraded a project I built for learning purposes from react_on_rails version 7 to version 8.0.0 and noticed that when deploying to Heroku, webpack no longer generates a minified production build of React (I can tell because I'm using the react extension for Chrome).

I upgraded using the generator from scratch and moving my code to the newly generated /client folder.

The source code of my project can be found [here](https://github.com/serodriguez68/react-on-rails-chat).

This is also happening with the [www.reactrails.com](https://www.reactrails.com/) app on version 8.

Is this a bug? or am I missing something in the client/webpack.config.js file?

**Comment by justin808** [comment]
@serodriguez68 Good catch. 

In the case of the generator, we're generating the simplest possible

**Comment by robwise** [comment]
yeah use `-p` for the production script

**Comment by shikendon** [comment]
And I found the `devBuild` variable in webpack.config.js always be `true` because of issue #877. It should be able to generate minified js file correctly after fix that.

**Comment by serodriguez68** [comment]
Ideally the generator should produce a client/package.json file that is ready to go for production builds.  I'm not familiar with react_on_rails codebase so I'm not sure where to make the change to make this happen.

Does doing the following change on [this file](https://github.com/shakacode/react_on_rails/blob/master/lib/generators/react_on_rails/templates/base/base/client/package.json.tt) makes the trick?

Change this: 
```
"build:production": "NODE_ENV=production webpack --config webpack.config.js",
```

For this:
```
"build:production": "NODE_ENV=production webpack -p --config webpack.config.js",
```

**Comment by justin808** [comment]
@serodriguez68 💯 

Please submit a PR and I'll merge it!

Even better if there's a unit test!

Also, please see https://github.com/shakacode/react_on_rails/blob/master/CONTRIBUTING.md for some basic PR hygiene like updating the https://github.com/shakacode/react_on_rails/blob/master/CHANGELOG.md

**Comment by serodriguez68** [comment]
Ok I just submitted [PR 893](https://github.com/shakacode/react_on_rails/pull/893) that fixes that.

**Comment by justin808** [comment]
Fixed in #895.
